libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -980,6 +982,33 @@ void AP_Logger_File::io_timer(void)
         write_fd_semaphore.give();
         return;
     }
+
+#if AP_FILESYSTEM_LITTLEFS_ENABLED
+    // see https://github.com/littlefs-project/littlefs/issues/564#issuecomment-2363032827
```

Comment:
it would be nice for this to be inside the AP_Filesystem LittleFS code somehow, could even be a static function on that class?
or just return short from the write() ?

Reply:
Fixed